### ISSUE TITLE
[BUG] Firefox screenshot does not show input values

### ISSUE THREAD

**Issue by aesyondu** [microsoft/playwright#1471]
**Context:**
- Playwright Version: next: https://github.com/microsoft/playwright/commit/535b4846371c00ebd5c763d6fa17aa26516a82bd
- Operating System: Mac OS X 10.14.6
- Extra: node@13.11.0, npm@6.13.7

**Code Snippet**

See *keyboard.spec.js*:
https://github.com/aesyondu/playwright/commit/dc90412b2e660cca4ced3c9ac221e7a0c2d05fda

**Describe the bug**

I was investigating an issue with `page.type` race conditions but discovered a different issue for firefox. In the above fork commit, I setup a test where:
* given a number of text inputs
* when text is typed into the inputs
* when screenshot is taken before and after typing
* then screenshot after typing should show the value typed to inputs

However, for firefox (`npm run funit`), the values do not show. See *zAfter.png*

**Comment by aesyondu** [comment]
Found here that values also do not appear in non-headless mode: https://github.com/aesyondu/bug-playwright-screenshot/commit/2de9dbf62c9ae00030019db881a2bcec290c9653

**Comment by aslushnikov** [comment]
@aesyondu can you please try with Playwright v0.13.0? I believe it should be fixed now.

**Comment by aesyondu** [comment]
I've tested in v0.13.0 (https://github.com/aesyondu/playwright/commit/a1abd2e60ea70ef6c6f0ce88b55749cebc7fc882) and in current master HEAD (https://github.com/aesyondu/playwright/commit/e5fe60bb3b947e0fde58d7cbde1ec5ede903b922). Firefox inputs still don't show.

I've also added headful test so it can be seen more clearly.

Steps:
```
git checkout a1abd2e # or e5fe60b
npm install && npm build
npm run funit # see in zAfter.png or in headfullzAfter.png that input values ARE NOT visible
npm run cunit # see in zAfter.png or in headfullzAfter.png that input values ARE visible
```

This is weird though, since I do have tests in my application that involve input boxes, and they show up fine in firefox. Perhaps this bug is caused by how I setup the input elements (createElement, body.append).
